src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -1167,7 +1167,8 @@
 									Grid.Row="2"
 									Grid.Column="0"
 									VerticalAlignment="Center"
-									Text="{x:Bind Commands.LayoutAdaptive.Label}" />
+									Text="{x:Bind Commands.LayoutAdaptive.Label}"
+									Visibility="{x:Bind Commands.LayoutAdaptive.IsExecutable, Mode=OneWay}" />
```

Comment:
Can you use `x:Load`?

Reply:
That doesn't work

Reply:
Can you confirm if it worked after recompiling the project? HotReload doen't always work for this type of change.

Reply:
Will try